src/Core/Commander/Providers/Fetcher.js

```diff
@@ -0,0 +1,41 @@
+import { TextureLoader } from 'three';
```

Comment:
I'm with @autra on this one.
I prefer to take full advantage of our chosen framework (three.js). We already have way bigger dependencies (Mesh, Materials, Vectors) so I don't see the value of *not* using the `TextureLoader` function.
Actually I'd rather drop our limited reimplementation of three.js features: `Camera` and `Scene` for instance but that's another story :)